PocketKit/Sources/Analytics/Events/ContentOpen.swift

```diff
@@ -35,6 +35,10 @@ public struct ContentOpen: Event {
         self.extraEntities = extraEntities
     }
 
+    public var description: String {
+        self.uiEntity.identifier
```

Comment:
This is what will be printed in logs when we log this object with `String(describing: )`